CCPA/Data Deletion Request Handling.md

```diff
@@ -0,0 +1,328 @@
+# Data Deletion Request Handling
+
+## Summary
+
+This specification provides a way for a publisher to communicate to downstream parties via a webpage about a consumer's request to delete their data. 
+
+In particular, Section 1798.105(c) of the California Consumer Protection Act (CCPA) states that "[a] business that receives a verifiable consumer request from a consumer to delete the consumer's personal information [shall] . . direct any service providers to delete the consumer's personal information from their records." 
+
+In response to this directive in the CCPA, this specification offers the technical means for publishers to communicate deletion requests to their service providers. All service providers covered by IAB Privacy LLC's Limited Service Provider Agreement (LSPA) and any other providers who receive the signal for deletion requests are then enabled to comply and delete the specified consumer data.
+
+While this specification provides the technical means for compliance in cases where CCPA applies, implementations may exist to support deletion requests outside CCPA governance. 
+
+
+### Relevant Documents
+
+[Limited Service Provider Agreement](https://www.iabprivacy.com/lspa-2019-12.pdf) (12-2-19 version)
+
+
+## How it Works
+
+Any ad tech company (vendor) supporting the US Privacy framework, exposes a JavaScript resource for each service they offer. The vendor hosts a URL that provides this resource and also serves as a unique identifier for the resource. The URL is made public to a maintained LSPA signatories list and to non-signatories based on the vendor's documented practices.
+
+Publishers working with any services provided by these vendors consume the exposed JavaScript and embed it on their properties where consumers can request deletion of their personal data. If data deletion is requested, all javascript signals on the publisher property are sent to the associated vendors. Vendors then respond by deleting the relevant personal data and signaling any affected vendors downstream.
+
```

Comment:
"...and signaling any affected vendors downstream."
I thought we were agreed that this was a one-level delete and not some kind of recursive fan out. Consider removing?